### PR TITLE
Access Conditional logs added

### PR DESCRIPTION
**Pull request by arafatalim**
Please review the following changes

- [x] Added conditional logs.

Logs
```json

localhost:3000/login-2023-08-14 09:58:14 {
  "timestamp": "2023-08-14 09:58:27",
  "status_code": 200,
  "http_method": "POST",
  "url": "localhost:3000/login",
  "hostname": "localhost:3000",
  "protocol": "http",
  "source": {
    "ip": "127.0.0.1",
    "user_agent": "PostmanRuntime/7.32.3"
  },
  "request_payload": {
    "type": "mobile_number",
    "value": "+918282985207",
    "password": "1111"
  },
  "response": "{\"access_token\":\"eyJhbGciOiJFZERTQSJ9.eyJleHAiOjE3MjM1NDQ5MDcsInN1YiI6IjdmYmMyYmNkLTgxMTctNDMyYi1hMGJjLTZjOGM5OTA1MGRhMyIsImF1ZCI6ImFjY2VzcyIsImlzcyI6InVzZXIiLCJqdGkiOiIzMzg0ZWFkYi04OWY1LTQ5ZjMtYTM3MS1lM2VmYTdjNjM1MjQifQ.QXRElDNEmn_IdQaXGpdXEeor9VxkNP3CCsZ8PQtJPYz3w1uWypTWv7wEo43W2pJcVPYCEhqJCFda6FYzvQ3_CQ\"}",
  "meta": {
    "id": "req-1"
  }
}

```

For more details regarding the above task, please visit the mars issue [#1206](https://mars.betalectic.com/apps/product-radical/radical/-/issues/1206)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -23,6 +34,43 @@ server.addHook("onRequest", async (req, reply) => {
   };
 });
 
+server.addHook("onSend", function (request, reply, payload, next) {
+  if (process.env.ENABLE_REQUEST_LOGGING === "true") {
```
Comment (rajeshn95) [inline server.js:38]: @arafatalim Please add this variable to `.env.example`

File: server.js [hunk 2]
```diff
@@ -15,6 +15,17 @@ const server = httpServer({
     }
   },
 });
+function getFormattedTimestamp() {
```
Comment (rajeshn95) [inline server.js:18]: @arafatalim Why this time is formatted like this, why cant we use `new Date().toISOString()` to get the iso date format?

File: server.js (hunk 1 above)
Comment (arafatalim) [reply server.js:38]: ok @rajeshn95

File: server.js (hunk 2 above)
Comment (arafatalim) [reply server.js]: removed the following function, now using `new Date().toISOString()`.

File: server.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ const server = httpServer({
     }
   },
 });
+const timestamps = new Date().toISOString();
```
Comment (rajeshn95) [inline server.js:18]: @arafatalim Once server is started, this time will be set and it won't change as it is defined as constant, so for every api call same time will be logged, please change it.

**Review by rajeshn95 — CHANGES_REQUESTED** [review]

File: server.js (hunk 3 above)
Comment (arafatalim) [reply server.js]: done!

**Review by rajeshn95 — APPROVED** [review]

**Pull request merged**